### ISSUE TITLE
question about data objects' key in hw2

### ISSUE THREAD

**Issue by yeppin** [swsnu/swppfall2021#41]
I have a question regarding hw2.
Can we assume that there are only two genders(male and female) in data objects?
I also wonder that whether we can assume the order of csv text (year, rank, name, gender, rankChange).

I will look forward to your reply, thank you.

**Comment by yeppin** [comment]
I found the answer of my second question in #26 (comment).

**Comment by kdh0102** [comment]
> Can we assume that there are only two genders(male and female) in data objects?

Yes, you can assume that only two genders appear in the data objects.
